.gitignore

```diff
@@ -1,5 +1,5 @@
+lib-test
 dist
-lib
```

Comment:
I think eventually we will have to do that as master is pretty much the development branch so better have people use from npm/bower? 

I think our deploy script in Vega-Lite and CompassQL could serve as a good start.